tests/test_ble_interface_core.py

```
@@ -4971,6 +4991,239 @@ def test_find_device_direct_connect_without_discovery_manager() -> None:
+        BLEInterface._invoke_safe_execute_compat(
+            safe_execute,
+            _handler,
+            error_msg="notification error",
+            fallback=_fallback,
+        )
+
+        assert len(handler_runs) == expect_handler_runs
+        assert len(fallback_runs) == expect_fallback_runs
+
+    _run_scenario(
+        lambda func, *args, **kwargs: func(),
```

Ruff (0.15.6)

[warning] Unused lambda argument: `args`

(ARG005)